### DISCUSSION TITLE
Whether this helm chart works for both Linux and Windows kubernetes clusters in AKS?/

### DISCUSSION THREAD

**Discussion by Hipersys2022** [signalfx/splunk-otel-collector-chart#1319] [General] [answered]
Hi Team,

im new to helm and Splunk HEC.

May you please let me know
Whether this helm chart works for both Linux and Windows kubernetes clusters in AKS?

i look forward for your response

**Comment by jinja2** [comment, marked as answer]
Yes - the chart supports both Linux and Windows Kubernetes clusters, including AKS with mixed node pools.

To deploy on Windows nodes, set `isWindows: true` in your Helm values. There's a ready-made example in the repo: [examples/kubernetes-windows-nodes](https://github.com/signalfx/splunk-otel-collector-chart/tree/main/examples/kubernetes-windows-nodes).

For a typical AKS cluster with both Linux and Windows node pools, you would deploy the chart twice - once for Linux nodes (default) and once for Windows nodes (with `isWindows: true`). The Windows deployment adjusts node selectors, disables hostNetwork (unsupported on Windows), and configures appropriate OS-specific settings.
